var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -51,8 +53,12 @@ def install_options(self, spec, prefix):
         args = [
             "--parallel={0}".format(make_jobs),
             "--ignore-git",
+            # "--verbose-build",
```

Comment:
Can delete if you don't want to keep

Reply:
left it there so i don't have to hunt for the switch to turn it on if necessary